### ISSUE TITLE
Server Crashing at starting (1.0.7)

### ISSUE THREAD

**Issue by K0stek15** [AllTheMods/ATM-8#824]
[latest.log](https://github.com/AllTheMods/ATM-8/files/10551441/latest.log)
Hello, i have a problem trying to start a server at this modpack.
It works fine yesterday, but today we cannot start it, even formating hosting and adding clear modpack and deleting world, won't worked. Anyone have some ideas what might help ?

**Comment by Mitchell5200** [comment]
Looks like you're using the wrong version of forge, the one for 1.0.7 is Forge 43.2.3, use that and then report back on any other issues that arise

**Comment by K0stek15** [comment]
Nope, original files have forge 43.1.55 , after uploding clean config 5 times it finally started working.
